tests/test_fe_utils.py

```diff
@@ -43,6 +43,36 @@ def test_extract_delta_Us_from_U_knk():
     np.testing.assert_almost_equal(expected_delta_Us, test_delta_Us)
 
 
+def test_image_frame():
+    rng = np.random.default_rng(2022)
+
+    coords = rng.random((90, 3))
+
+    max_dimensions = np.max(coords, axis=0)
+
+    # Add a random buffer to the dimensions of the box
+    box = np.eye(3) * (max_dimensions + rng.random(max_dimensions.shape))
+    group_indices = np.arange(len(coords)).reshape((-1, 3))
```

Comment:
nit: idea for strengthening test: group of different sizes, including some empty?

Reply:
Strengthened in [3b58a0b](https://github.com/proteneer/timemachine/pull/905/commits/3b58a0ba366ae6c2d9a7da2446961aa52271ccee)